source/ai/rl/model/breakout_bbf.py

```
@@ -0,0 +1,212 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+import numpy as np
+
+
+class MLP(nn.Module):
+    def __init__(self, in_hiddens=512, med_hiddens=512, out_hiddens=512, layers=1,
+                  act=nn.ReLU(), out_act=nn.Identity(), bias=True):
```

Ruff (0.12.2)

Do not perform function call `nn.ReLU` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)